pkg/cli/store.go

```diff
@@ -70,6 +70,9 @@ func init() {
 	// the ingestor will query and ingest endoflife.date for EOL
 	set.Bool("add-eol-on-ingest", false, "if enabled, the ingestor will query and ingest endoflife.date for EOL data. Warning: This will increase ingestion times")
 
+	// the ingestor will query and ingest deps.dev data for scorecards and source association
```

Comment:
Missing the flag here on `guaccollect`: https://github.com/guacsec/guac/blob/main/cmd/guacingest/cmd/root.go#L34

and `guacone`: https://github.com/guacsec/guac/blob/main/cmd/guacone/cmd/root.go#L34